bookstore/booksauthor/functions.py

```diff
@@ -23,5 +23,8 @@ def saveBook(validated_data):
     if not serializer.is_valid():
         return serializer.errors, False
     serializer.save()
-    print(serializer)
-    return serializer, True
\ No newline at end of file
+
+    return True, serializer
+
+    return serializer, True
+
```

Comment:
there are two line, remove "True, serializer". This function or any other function where we serialize should return 
(serializer, bool)